src/Game.cpp

```diff
@@ -849,7 +850,7 @@ bool Game::CanLoadGame(const std::string &filename)
 {
 	Output("Game::CanLoadGame('%s')\n", filename.c_str());
 	auto file = FileSystem::userFiles.ReadFile(FileSystem::JoinPathBelow(Pi::SAVE_DIR_NAME, filename));
-	if (!file) 
```

Comment:
Here and below there are several whitespace/line changes that just add noise to the review making it harder to do.